molecule/default/molecule-no-systemd.yml

```diff
@@ -19,6 +19,20 @@ platforms:
   - name: debian9
     image: debian:stretch-slim
     dockerfile: Dockerfile_debian_9.j2
+  # - name: debian10
+  #   image: debian:buster-slim
+  # - name: debian11
+  #   image: debian:bullseye-slim
+  # - name: kali
+  #   image: kalilinux/kali-rolling
+  # - name: fedora32
+  #   image: fedora:32
+  # - name: fedora33
+  #   image: fedora:33
+  # - name: ubuntu18
+  #   image: ubuntu:bionic
+  # - name: ubuntu20
+  #   image: ubuntu:focal
```

Comment:
Should we add a comment here to explain why all of these other platforms are commented out?  Otherwise, we I think should just remove them entirely like we do here:
https://github.com/cisagov/ansible-role-cyhy-commander/blob/2a59c7a94bec0443615ee65c05474df1440f8066/meta/main.yml#L16-L19

Reply:
I can add a comment. I was trying to align with [cisagov/ansible-role-cyhy-core](https://github.com/cisagov/ansible-role-cyhy-core), though that has a comment so I failed a bit there.

Reply:
Thanks- that comment should be added in all of the other similar lineage PRs.

Reply:
Added a comment in 47aacec. Please let me know your thoughts and I will add a comment similar to the one finalized here to the other PRs.

Reply:
That comment looks great to me, thanks for adding it!